Scripts/Mac/LayoutFunctionalTests.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+. "$(dirname ${BASH_SOURCE[0]})/InitializeEnvironment.sh"
+
+CONFIGURATION=$1
+LAYOUT_ROOT=$2
```

Comment:
I think the meaning of "layout" will become fuzzy very quickly. This script is just creating a build drop that can be published from one build and then consumed from another, right?

How about the name `CreateBuildDrop.sh` for the script name, and `BuildDrop` anywhere you use `Layout` inside the script?

I also didn't find an equivalent script for Windows, how is it being created there?

Reply:
`BuildDrop` makes sense. I'll retool this.

I'm going to create a similar script for Windows. I kept it out of this PR in the interest of focus, so I don't have to hop between two boxes. It will the new PR I post. :)